### ISSUE TITLE
Process planning doc into tickets

### ISSUE THREAD

**Issue by mikelmaron** [MapsCamp/MapsCamp#3]
https://docs.google.com/document/d/1rj_f-l0bS9z9V2g_8ALtQdRpxPN2yIu48TUCc-yYUpY/edit

**Comment by willykaram** [comment]
Should we possibly try to update/revise the exisiting collaboration/planning document:

https://docs.google.com/document/d/1cC0bIG01wNfWAkzV6sjZJrufQ0QtytAYYkDWGXnxUSY/edit

Or do you think it would be better/easier to start with something fresh/more focused, and archive the old one?

**Comment by awright** [comment]
I interpret this ticket as translate the planning document into actionable tickets. Could be misunderstanding here though.

**Comment by mikelmaron** [comment]
@awright @willykaram exactly. didn't want to lose anything from the planning docs as we start taking actions....

**Comment by mikedillion** [comment]
@mikelmaron That Google doc with all the passwords is in the open!
